cpp/modmesh/view/RManager.cpp

```diff
@@ -117,57 +117,46 @@ void RManager::setUpMenu()
     // NOTE: All menus need to be populated or Windows may crash with
     // "exited with code -1073740791".  The reason is not yet clarified.
 
+    m_fileMenu = m_mainWindow->menuBar()->addMenu(QString("File"));
     {
-        m_fileMenu = m_mainWindow->menuBar()->addMenu(QString("File"));
-
-        {
-            auto * action = new RAction(
+        m_fileMenu->addAction(
+            new RAction(
                 QString("New file"),
                 QString("Create new file"),
                 []()
                 {
                     qDebug() << "This is only a demo: Create new file!";
-                });
-            m_fileMenu->addAction(action);
-        }
+                }));
 
-        m_eulerMenu = m_mainWindow->menuBar()->addMenu(QString("Euler"));
+        m_fileMenu->addAction(new RAction(
+            QString("Parameters"),
+            QString("Runtime parameters"),
+            []()
+            {
+                static int64_t int64V = 5566;
+                static double doubleV = 77.88;
+                auto params = createParameters();
+                addParam(params, "global.a.b.int64_foo", &int64V);
+                addParam(params, "global.a.b.double_bar", &doubleV);
+                openParameterView(params);
+            }));
 
-        {
-            auto app_name = QString("euler1d");
-            auto * action = new RAppAction(
-                QString("Load ") + app_name,
-                QString("Load ") + app_name,
-                QString("modmesh.app.") + app_name);
-            m_eulerMenu->addAction(action);
-        }
 #ifndef Q_OS_MACOS
```

Comment:
It will be better if we can use a runtime check for adding the quit/exit menu item when it is not macos.  It should be a future enhancement.